### PR TITLE
New paths UI sankey card

### PR DESCRIPTION
**Pull request by liyiy**
## Changes

*Please describe.*  
*If this affects the frontend, include screenshots.*  

An update to our current paths visualization where we've added cards that display more information about each path node.

Screen Shot 2021-09-23 at 5 40 44 PM

https://user-images.githubusercontent.com/25164963/134455503-2066e2d8-1fca-4575-a102-46f21e484b88.mov

## Checklist

- [ ] All querysets/queries filter by Organization, by Team, and by User
- [ ] Django backend tests
- [ ] Jest frontend tests
- [ ] Cypress end-to-end tests
- [ ] Migrations are safe to run at scale (e.g. PostHog Cloud) – present proof if not obvious
- [ ] New/changed UI is decent on smartphones (viewport width around 360px)

### CODE REVIEW

**Review by clarkus — CHANGES_REQUESTED** [review]
Nice progress! Is there anything we can do to make the path item metrics feel more connected to the path name / url? There's some space and rounding separating the items in the examples. It almost feels like a sub-menu, and really it's a fully revealed state of a single item. 

Screen Shot 2021-09-23 at 2 56 48 PM

**Comment by liyiy** [comment]
> Nice progress! Is there anything we can do to make the path item metrics feel more connected to the path name / url? There's some space and rounding separating the items in the examples. It almost feels like a sub-menu, and really it's a fully revealed state of a single item.
> 
> Screen Shot 2021-09-23 at 2 56 48 PM

I can try to make it cleaner around the edges but I'm using antd's dropdown for this to avoid writing a whole dropdown component 😅 that's why it looks like a sub menu heh

**Comment by liyiy** [comment]
update: did some finagling with the border radii and margin tops, is this better? @clarkus 

Screen Shot 2021-09-24 at 2 48 04 AM

**Comment by clarkus** [comment]
> update: did some finagling with the border radii and margin tops, is this better? @clarkus
> 
> Screen Shot 2021-09-24 at 2 48 04 AM

Yeah it's definitely better! If the path item name portion were the same color as the rest of the card and had the same borders, I think it'd match the design completely.

**Comment by liyiy** [comment]
yes!! good catch @clarkus Screen Shot 2021-09-26 at 4 26 34 AM

File: frontend/src/scenes/paths/Paths.js [hunk 1]
```diff
@@ -261,6 +234,132 @@ export function Paths({ dashboardItemId = null, filters = null, color = 'white'
         >
             <div ref={canvas} className="paths" data-attr="paths-viz">
                 {!pathsLoading && paths && paths.nodes.length === 0 && !paths.error && <NoData />}
+                {!paths.error &&
```
Comment (EDsCODE) [inline frontend/src/scenes/paths/Paths.js:237]: Maybe move these inline styles to the new scss file you created?

**Review by EDsCODE — CHANGES_REQUESTED** [review]
1. Dropdown onHover shows a cursor but clicking doesn't do anything
2. The path cards at the end are top aligned whereas all the other cards are centered vertically on the node
Screen Shot 2021-09-27 at 9 23 21 AM

Follow up design considerations from my testing:
1. would be helpful to squeeze some number onto the cards (without dropping down)
2. I think we need a bit more in the label "average conversion time" that explains what the time actually is. Right now I can't tell if it's the average time it took people to get to the node or leave the node or spent at the node

Overall, the path cards look awesome. They're already way more fun to use and look better than the simple labels we had before

**Comment by liyiy** [comment]
@EDsCODE  The path cards at the end are top aligned otherwise they'll intersect each other, do you have suggestions for alternatives?

**Comment by clarkus** [comment]
> @EDsCODE The path cards at the end are top aligned otherwise they'll intersect each other, do you have suggestions for alternatives?

This is pretty directly related to the number of steps shown consecutively and the width of the path item labels. In the designs, when they're not hovered, we just show the path. This keeps the card succinct and reduces the risk over conflicting with other cards. That doesn't prevent this happening entirely, but it does a lot to mitigate it. 

One option I tried in the most recent version was increasing the width of the item columns such that there is always sufficient space to show a card that flows left-to-right. This means that cards should always have space to the right.

Screen Shot 2021-09-27 at 9 50 52 AM

**Comment by EDsCODE** [comment]
> @EDsCODE The path cards at the end are top aligned otherwise they'll intersect each other, do you have suggestions for alternatives?

That makes sense. My first reaction was just that it seemed so far pushed to the top but I think the spacing against the other cards makes sense so should be fine.

**Review by EDsCODE — CHANGES_REQUESTED** [review]
Can you actually feature flag this even though changes should work with current api? I just want us to go through testing phases and make sure there aren't crazy edge cases we're missing that might come with prod

After the feature flag, should be good to merge

**Review by EDsCODE — APPROVED** [review]
good to go! will merge with it behind a feature flag

**Pull request merged**